k8s-dev/cstor-pool-mgmt/pool-resource-cluster-1.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: openebs.io/v1alpha1
+kind: CStorPool
+metadata:
+  name: pool1-abc
+  node: node-host-label
+  labels:
+#The following label will be used for additional node reference, since we go for UID based selection of pod.
+    "kubernetes.io/hostname": "node-host-label"
+
+#finalizer helps handling deletion of resource
+  finalizers: ["cstorpool.openebs.io/finalizer"]
+
+spec:
+  disks:
+   diskList: ["/dev/sdc1"]
+  poolSpec:
+   cacheFile: /tmp/pool1.cache
```

Comment:
will it be advisable to move all the three poolSpec properties to directly under spec:?

Reply:
As discussed, this diskList([ ] string) will be changed to disk([ ] struct) in another pull request. and we will continue to have separate fields for disks and poolspec for segragating and adding more disk related info(spare disk, raid info, etc).